mne/export/_edf.py

```diff
@@ -198,10 +198,15 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
                     raise RuntimeError(
                         f"Setting patient birth date to {birthday} "
                         f"returned an error")
+
```

Comment:
```suggestion
```
The easiest way for me to get all CIs to run is to make a trivial commit, so here we go!